tesseract_visualization/test/trajectory_player_unit.cpp

```diff
@@ -52,7 +52,7 @@ TEST(TesseracTrajectoryPlayerUnit, TrajectoryTest)  // NOLINT
   TrajectoryPlayer player;
   player.setTrajectory(trajectory);
 
-  EXPECT_NEAR(player.trajectoryDuration(), 9, 1e-5);
```

Comment:
Probably be good to a test for the trajectoryDurationStart() method.

Reply:
That's a good one. Will do tomorrow